projects/ng-material-multilevel-menu/src/lib/list-item/list-item.component.html

```diff
@@ -1,25 +1,27 @@
 <mat-list-item matRipple [matRippleDisabled]="node.disabled" [ngClass]="selectedListClasses" *ngIf="!node.hidden"
-  (click)="expand(node)" title="{{node.label}}"
-  [ngStyle]="getListStyle()">
-  <div class="anml-data" [dir]="isRtlLayout() ? 'rtl' : 'ltr'">
-    <div class="icon-container" [ngSwitch]="getListIcon(node)">
-      <span *ngSwitchCase="'faicon'" class="amml-icon amml-icon-fa">
-        <i [ngClass]="getSelectedFaIcon()"></i>
-      </span>
-      <mat-icon *ngSwitchCase="'icon'" class="amml-icon">
-        {{getSelectedIcon()}}
-      </mat-icon>
-      <mat-icon *ngSwitchCase="'svgicon'" svgIcon="{{getSelectedSvgIcon()}}" class="amml-icon amml-svg-icon">
-      </mat-icon>
-      <img matListAvatar *ngSwitchCase="'imageicon'" class="amml-icon" src="{{getSelectedImageIcon()}}" alt="{{node.label}}"/>
+  (click)="expand(node)" title="{{node.label}}" [ngStyle]="getListStyle()">
+  <a [routerLink]="node.link" class="menu-link">
```

Comment:
The routerLink will give the wrong URL if the URL is for another website. For example, it will give URL like this,
` http://localhost:4200/https:/www.npmjs.com/package/ng-material-multilevel-menu`. So I think the approach needs to change here.

In the below image check the left-bottom corner, where you will see the URL.
![image](https://user-images.githubusercontent.com/15548024/79968011-34e73e00-84ad-11ea-9608-ffb8df7af0a3.png)

Reply:
I solved this adding a new component named "link".